### DISCUSSION TITLE
Xdebug - debugging issue

### DISCUSSION THREAD

**Discussion by boan-jfm** [libvips/php-vips#224] [Q&A]
Hi

I am trying to debug my program, and when the execution encounters parts of the [ImageAutodoc](https://github.com/libvips/php-vips/blob/14df89980b4186209bbfe6e920fceec817b6ef8f/src/ImageAutodoc.php#L704) (extended by [Image](https://github.com/libvips/php-vips/blob/14df89980b4186209bbfe6e920fceec817b6ef8f/src/Image.php)) method calls, it stops with an error: 'Process finished with exit code 11.' Any idea how to avoid this, maybe by just ignoring the calls? I imagine the issue is with the FFI calls where the debugger can't follow the execution.

**Comment by jcupitt** [comment]
You'll need to give more information. Which debugger, what version, what program, how did you run it, what platform, etc. etc.

**Comment by boan-jfm** [comment]
Windows 10 - 22H2 - build: 19045.3448
WSL-version: 1.2.5.0
WSL-kernelversion: 5.15.90.1
WSL distribution version: 2

Ubuntu in WSL2:
Description:    Ubuntu 22.04.3 LTS

PHP:
PHP 8.1.2-1ubuntu2.14 (cli) (built: Aug 18 2023 11:41:11) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.2-1ubuntu2.14, Copyright (c), by Zend Technologies
    with Xdebug v3.1.2, Copyright (c) 2002-2021, by Derick Rethans

IDE:
PhpStorm 2023.2.2
Build #PS-232.9921.55, built on September 15, 2023

Example:
![image](https://github.com/libvips/php-vips/assets/13201214/02536559-57f9-422d-8dba-6e1a761be924)
The breakpoint is working fine at line 13, but stepping over the `$image->colourspace...` (line 14) results in a exit code 11
